### ISSUE TITLE
TODO: Shuttle Console sprites B: 50

### ISSUE THREAD

**Issue by DooblyNoobly** [unitystation/unitystation#1074]
## TODO
Design and create the sprites that will be used on the shuttle console pop up window. The shuttle console will be used to pilot shuttles and reposition stations. See the proposed layout below:

![image](https://user-images.githubusercontent.com/20813925/40263443-ba958e40-5b55-11e8-941e-cea0a1bbfb3d.png)

## Criteria:

 1. Has to have a radar screen or mini map to see the positions of other stations/shuttles/asteriods near by
 2. A thruster slider or speed slider
 3. Start/Stop button
 4. Left and right keys to manually steer the shuttle
 5. Waypoint graphics (the pilot can click on the minimap to place waypoints so create the waypoint pin graphics that will appear when the pilot clicks the screen)
 6. Create the other minimap graphics that you think will be needed in the development of the shuttle console (other shuttle markers, station markers, asteriods etc)

Contributor who meets all of the criteria above will be rewarded with the 50usd bounty (pending a review from the heads).

**Comment by NinjaDanz3r** [comment]
First pass at this, feedback appreciated.
Anything can be recolored as needed.
As for text and stuff like that, I propose that we use Unity UI labels on top of the sprite, in order to make the workflow of adding new buttons easier, as baking that into the sprite would require further editing to be made in the future.
https://imgur.com/a/Es6pDhu

**Comment by DooblyNoobly** [comment]
Looking good so far. Yep UI.Text components will be used for all text in UI. Would you be able to create a mockup with all these sprites laid out so we can get a better idea of constrast etc?

**Comment by NinjaDanz3r** [comment]
Sure! Will post again when it's done.

**Comment by NinjaDanz3r** [comment]
Very quick mockup:
https://imgur.com/a/2qnDngh

**Comment by fomalsd** [comment]
These grey buttons still look weird, everything else seems to look fine for me.
Could you place them according to the layout above?

**Comment by NinjaDanz3r** [comment]
Here you go:
https://imgur.com/a/fb3TdJQ
Replaced square button with rounded buttons for now, but I can make another pass at the square buttons but this time make them in the same style as the rounded buttons this time, if you want to.

**Comment by dapocalypse** [comment]
Can we have something akin to baystation where every z level is on a coordinate plane

**Comment by fomalsd** [comment]
too early for z-levels.
maybe upscale these sprites 2x for the pixelated look?

**Comment by dapocalypse** [comment]
How many units should the minimap be(I am doing a grid)

**Comment by NinjaDanz3r** [comment]
@fomalsd I think you can upscale either the sprites or the UI graphics in the editor, so I don't think there's any need to upscale the originals? Or do you mean for the mockup?

**Comment by fomalsd** [comment]
yeah, for the mockup. I suppose 2x UI scale would be the standard one

**Comment by NinjaDanz3r** [comment]
@fomalsd scaled up sprites 2X, take a look: https://imgur.com/a/Ykb5Ssy .

**Comment by DooblyNoobly** [comment]
Looking good. Can you also put some mock up minimap indicators on the screen so we can get an idea of how that part of it will look/work.

**Comment by NinjaDanz3r** [comment]
Here, scaled and unscaled 16x16 icons for comparison.
https://imgur.com/a/DkdG3Qq

**Comment by DooblyNoobly** [comment]
Cool looks good. Can you put the sprites into a folder in the repo, slice them and make sure texture import settings are on full rect. Once that is done I'll close this bounty

**Comment by NinjaDanz3r** [comment]
Yep, will start working on that tomorrow.

**Comment by fomalsd** [comment]
I think these buttons need a little variety.
While red one would be suitable for start/stop toggle, maybe try drawing finger-friendly switches similar to the one attached below for autopilot and safety mode toggles?
And then maybe give the grey keyish ones another try, I think they will look great when used as L/R keys.
![images 01](https://user-images.githubusercontent.com/10403536/40291833-1b9caf86-5cd0-11e8-998d-c00b99ba7a62.jpg)

**Comment by NinjaDanz3r** [comment]
Oooooooh I recognize that button, it's from one of the Commander Keen games, 3 or 4 I believe.
I'll do another pass on the keyboard buttons then, and I'll gladly do a toggle switch like that.

**Comment by dapocalypse** [comment]
I think piloting a ship should be more complicated. Maybe add more buttons.

**Comment by dapocalypse** [comment]
Can i make button not listed.

**Comment by fomalsd** [comment]
@dapocalypse nope, that's not needed for the moment

**Comment by dapocalypse** [comment]
Can i change the lr, text?

**Comment by dapocalypse** [comment]
Left Right*
